controllers/tests/controller_server/host_definer_server_test.py

```diff
@@ -26,14 +27,14 @@ def setUp(self):
         self.storage_agent = MagicMock()
         mock_get_agent(self, HOST_DEFINER_SERVER_PATH)
 
-        self.request = Mock(spec_set=['prefix', 'connectivity_type', 'node_id', 'system_info'])
+        self.request = Mock(spec_set=['prefix', 'connectivity_type', 'node_id', 'array_connection_info'])
 
         self.iqn = 'iqn.1994-05.com.redhat:686358c930fe'
         self.nqn = 'nqn.2014-08.org.nvmexpress:uuid:b57708c7-5bb6-46a0-b2af-9d824bf539e1'
         self.request.prefix = None
         self.request.connectivity_type = 'fc'
         self.request.node_id = '{};;;{}'.format(HOST_NAME, self.iqn)
-        self.request.system_info = SECRET
+        self.request.array_connection_info = get_array_connection_info_from_secrets(SECRET)
```

Comment:
do we wanna use the actual function or mock it?
what if there is an "error" in the function?
@ELENAGER what do you think?

Reply:
we are not testing function get_array_connection_info_from_secrets, so, I think we need to use a mock

Reply:
done